docker/dev/tls/docker-compose.override.example.yml

```diff
@@ -3,8 +3,7 @@ services:
     volumes:
       # Linux: mount the certificate bundle
       - /etc/ssl/certs/ca-certificates.crt:/etc/ssl/certs/ca-certificates.crt:ro
-
-  step:
-    # add any host here run in a docker service, that need a TLS certificate
-    extra_hosts:
-      - "openproject.local:host-gateway"
+    networks:
+      external:
+        aliases:
+          - openproject.local
```

Comment:
Can you shortly elaborate, why this alias has to be `openproject.local`? I mean, I know it, but for documentation sake we could add a comment here, telling, that for TLS validation step-ca needs to go through traefik to reach the correct target.

Nice find! 👍🏽

Reply:
Done, good idea :+1: